iroha_client/src/client.rs

```diff
@@ -18,7 +17,7 @@ pub struct Client {
 impl Client {
     pub fn new(configuration: &Configuration) -> Self {
         Client {
-            torii_url: configuration.torii_url.clone(),
+            torii_url: configuration.torii_api_url.clone(),
```

Comment:
I think clients need only api_url so in client::config::Configuration we may still have torii_url.

Reply:
Yes, they need api_url, and by naming it here like that we show them which of the urls to take from peer config. I think it is better to have this naming - it gives a hint to the users.

Reply:
![image](https://user-images.githubusercontent.com/5635032/92090767-e5ad0480-ede0-11ea-8004-56f90c101dc4.png)